src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1610,9 +1610,12 @@ public void BuildBasicApplicationCheckMdbAndPortablePdb ()
 		[Test]
 		public void BuildBasicApplicationCheckConfigFiles ()
 		{
+			if (Builder.UseDotNet) {
+				Assert.Ignore ("'.dll.config' files are not supported in .NET 5+.");
```

Comment:
Should this test be ignored under `dotnet`?  Or should it instead assert that the resulting `.apk` file doesn't contain `.dll.config` files?

Additionally, there is the [migration question](https://github.com/xamarin/xamarin-android/pull/4896#issuecomment-654944896): if an existing project contains a `.dll.config` file, should *anything else* be done in addition to "ignoring" the presence of the file?  Should we emit a diagnostic message, or a warning, or *anything*?

If we *do* decide to "do something", that "something" could be asserted in this test.

Reply:
I added a `XA1024` warning for any `.config` file found in `@(ResolvedFileToPublish)`.